tests/modules/programs/scmpuff/no-fish.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, lib, ... }: {
+  config = {
+    programs = {
+      scmpuff = {
+        enable = true;
+        enableFishIntegration = false;
+      };
+      fish.enable = true;
+    };
+
+    # Needed to avoid error with dummy fish package.
+    xdg.dataFile."fish/home-manager_generated_completions".source =
+      lib.mkForce (builtins.toFile "empty" "");
+
+    test.stubs.fish = { };
+
+    nmt.script = ''
+      assertFileNotRegex home-files/.config/fish/config.fish '${pkgs.scmpuff}'
+    '';
```

Comment:
Same thing about overriding the scmpuff package

Reply:
Done